docker-entrypoint.sh

```diff
@@ -40,7 +40,7 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 		group="$(id -g)"
 	fi
 
-	if ! [ -e index.php -a -e wp-includes/version.php ]; then
+	if ! [ -e index.php -e wp-includes/version.php ]; then
```

Comment:
This isn't quite right:

```console
$ [ -e xyz -e abc ]
-bash: [: too many arguments
```

What this needs to be updated to instead is something like:

```bash
	if [ ! -e index.php ] && [ ! -e wp-includes/version.php ]; then
```

Reply:
ACK

Reply:
Resolved.  Thanks for the review.